### PR TITLE
CLI and API added

### PR DESCRIPTION
**Pull request by Luckygyana**
- process_file_list function added which directly take files and store regression output in npz file
- __main__.py added for clean cli 
- Test for models.py
- Test for core.py
- Test for cli.py

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    assert np.all(np.abs(ts1load - ts1) < tol)
+
+    # Make sure that the embeddings are approximately the same with mono and stereo
+    audio, sr = sf.read(filenames["CHIRP_STEREO_PATH"])
+    emb2, _ = torchopenl3.get_audio_embedding(
+        audio, sr, model=model, center=True, hop_size=0.1, verbose=True
+    )
+    emb2 = to_numpy(emb2)
+    # assert np.all(np.abs(emb1 - emb2) < tol)
+    # assert np.all(np.abs(ts1 - ts2) < tol)
```
Comment (khumairraj) [inline tests/test_core.py:248]: Do we need to comment these tests? For checking similarity between mono and stereo?

File: tests/test_core.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+    assert not np.any(np.isnan(emb2))
+
+    # Make sure that the embeddings are approximately the same if we resample the audio
+    audio, sr = sf.read(filenames["CHIRP_44K_PATH"])
+    emb3, _ = torchopenl3.get_audio_embedding(
+        audio, sr, model=model, center=True, hop_size=0.1, verbose=True
+    )
+    emb3 = to_numpy(emb3)
+    # assert np.all(np.abs(emb1 - emb3) < tol)
+    # assert np.all(np.abs(ts1 - ts3) < tol)
```
Comment (khumairraj) [inline tests/test_core.py:258]: same with the resampling.

Comment (Luckygyana) [reply tests/test_core.py:258]: We can do this. First I thought to check if test case are passing for others or not.

**Review by khumairraj — CHANGES_REQUESTED** [review]
Lets either Add the change which you made yesterday to remove the julian and make resampy as the default one or  make resampy as the default one in core.py, so that the tests pass. 
In the regression tests I have explicitly passed resampy, but anyways since we are going to remove the julius we should not do this explicit pass in cli and other tests and rather either do the change you did yesterday or make resampy as default.

File: tests/test_core.py (hunk 2 above)
Comment (khumairraj) [reply tests/test_core.py:249]: sure. i think that will work

**Comment by khumairraj** [comment]
> Lets either Add the change which you made yesterday to remove the julian and make resampy as the default one or make resampy as the default one in core.py, so that the tests pass.
> In the regression tests I have explicitly passed resampy, but anyways since we are going to remove the julius we should not do this explicit pass in cli and other tests and rather either do the change you did yesterday or make resampy as default.
Here [https://github.com/turian/torchopenl3/blob/9bd09578186fb3c13fd6d84b919d3a23e2b68f59/torchopenl3/core.py#L113](url)

**Pull request merged**